src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's Contact in the address book.
+ * Parent class of Phone, Email and Address classes which extract commonalities from child classes.
+ */
+public class Contact {
+
+    public String value;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.